Description: Fix non portable font
Author: Jörg Frings-Fürst <debian@jff.email>
Bug: https://gitlab.com/sane-project/backends/-/issues/738
Last-Update: 2024-03-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/doc/sane-hp5590.man
===================================================================
--- trunk.orig/doc/sane-hp5590.man
+++ trunk/doc/sane-hp5590.man
@@ -45,7 +45,7 @@ or
 Valid command line options and their syntax can be listed by using:
 .IP
 .nf
-\f[C]
+\f[I]
 scanimage --help -d hp5590:interface:device
 \f[R]
 .fi
@@ -63,7 +63,7 @@ on the first scanner identified.
 Use the command:
 .IP
 .nf
-\f[C]
+\f[I]
 scanimage -L
 \f[R]
 .fi
@@ -255,7 +255,7 @@ to be included in
 .I hp5590.conf
 .IP
 .nf
-\f[C]
+\f[I]
 device hp5590 {
     # Device matching
     filter = \[dq]\[ha]hp5590.*\[dq]
@@ -285,7 +285,7 @@ device hp5590 {
 \f[B]scan_action.script\f[R]
 .IP
 .nf
-\f[C]
+\f[I]
 #!/bin/bash
 echo device = $SCANBD_DEVICE
 echo action = $SCANBD_ACTION
@@ -319,7 +319,7 @@ variable controls the debug level for th
 Higher debug levels increase the verbosity of the output:
 .IP
 .nf
-\f[C]
+\f[I]
 10 - generic processing
 20 - verbose backend messages
 40 - HP5590 high-level commands
